### PR TITLE
ParseCSV - Added codec type support

### PR DESCRIPTION
**Pull request by yardensade**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/15925

## Description
ParseCSV would fail to decode iso-8859-8 character sets, added support to switch the codec type from utf-8 (default) to anything else.

## Screenshots
image
image

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [ ] Documentation
- [x] Code Review

## Additional changes
Describe additional changes done, for example adding a function to common server.

### CODE REVIEW

File: Scripts/script-ParseCSV.yml [hunk 1]
```diff
@@ -271,6 +271,10 @@ args:
   - "no"
   description: parses and converts all the rows in csv into json and puts into context.
   defaultValue: "yes"
+- name: codec
+  description: The codec type used to parse the file (some character sets are not
```
Comment (Itay4) [inline Scripts/script-ParseCSV.yml]: please let @kirbles19 review the description

File: Scripts/script-ParseCSV.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ script: |+
 
   reload(sys)
   sys.setdefaultencoding('utf8')
```
Comment (Itay4) [inline Scripts/script-ParseCSV.yml]: shouldn't we handle that too ?

Comment (yardensade) [reply Scripts/script-ParseCSV.yml]: irrelevant to the parsing action.

File: Scripts/script-ParseCSV.yml (hunk 1 above)
Comment (yardensade) [reply Scripts/script-ParseCSV.yml]: Contacted him now.

File: TestPlaybooks/playbook-TestParseCSV.yml [hunk 3]
```diff
@@ -110,30 +117,34 @@ tasks:
...
     type: regular
     task:
-      id: 7cb82eab-f88e-4602-8dae-9a590de588f1
+      id: a5bc2877-0009-4dcf-8006-8d83e5a22c7c
       version: -1
       name: ParseCSV tata.csv file
       description: This script will parse a CSV file and place the unique IPs, Domains
         and Hashes into the context.
-      scriptName: ParseCSV
+      scriptName: ParseCSV_copy_1
```
Comment (Itay4) [inline TestPlaybooks/playbook-TestParseCSV.yml]: remove `copy`

File: TestPlaybooks/playbook-TestParseCSV.yml [hunk 4]
```diff
@@ -168,27 +181,254 @@ tasks:
...
+    id: "9"
+    taskid: c6e4fa56-b7d3-4e9e-8578-b97310d6ddba
+    type: regular
+    task:
+      id: c6e4fa56-b7d3-4e9e-8578-b97310d6ddba
+      version: -1
+      name: ParseCSV iso.csv file
+      description: This script will parse a CSV file and place the unique IPs, Domains
+        and Hashes into the context.
+      scriptName: ParseCSV_copy_1
```
Comment (Itay4) [inline TestPlaybooks/playbook-TestParseCSV.yml]: remove `copy`

**Pull request merged**